src/Ledger/PParams.lagda

```diff
@@ -1,4 +1,9 @@
-\section{Protocol parameters}
+\section{Protocol parameters\protect\footnotemark}
+\label{sec:protocol-parameters}
+\footnotetext{See also: \href{https://github.com/cardano-foundation/CIPs/tree/master/CIP-1694}{github.com/cardano-foundation/CIPs/CIP-1694}}
+This module defines the adjustable protocol parameters of the Cardano ledger.
```

Comment:
We shouldn't talk about modules and imports in the prose, but about sections instead.